app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -94,6 +94,10 @@ def available_resource_pools(opts = {})
       resource_pools({ :cluster_id => opts[:cluster_id] })
     end
 
+    def available_images
+      client.templates
```

Comment:
it's slow, but not insanely. took 5-8 seconds, which is annoying, but not that bad.